qcodes/instrument/group_parameter.py

```diff
@@ -210,27 +204,27 @@ def parser(ret_str: str) -> Dict[str, Any]:
 
         return parser
 
-    def set(self, set_parameter: GroupParameter, value: Any) -> None:
+    def set(self, set_parameter: GroupParameter, raw_value: Any) -> None:
         """
         Sets the value of the given parameter within a group to the given
```

Comment:
I think this method is super confusing. It does not actually set all of the group and it takes a raw value which is confusing. I vote for deprecating it and replacing with a private method that does the same.

Reply:
wholeheartedly support!